clients/src/main/java/org/apache/kafka/common/security/kerberos/KerberosLogin.java

```diff
@@ -219,7 +219,7 @@ public void run() {
                                     }
                                 } else {
                                     log.warn("[Principal={}]: Could not renew TGT due to problem running shell command: '{} {}'; " +
-                                            "exception was: %s. Exiting refresh thread.", 
+                                            "exception was: {}. Exiting refresh thread.",
```

Comment:
I don't think we really need to include the exception in the message since we'll get the full trace anyway.